### PR TITLE
lambda.exercise1 (Turenko)

### PR DESCRIPTION
**Pull request by turenkoaa**

### CODE REVIEW

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 1]
```diff
@@ -57,20 +84,32 @@ public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator
     public void findFirstWithAge30UsingGuavaPredicate() {
         List<Person> persons = Arrays.asList(getPersons());
 
-        // TODO использовать FluentIterable
-        Person person = null;
+        Predicate<Person> isFirstNameAlexChecker = new Predicate<Person>() {
```
Comment (AleksandrRadchenko) [inline src/test/java/lambda/part1/exercise/Exercise1.java]: Predicate name irrelevant

**Review by AleksandrRadchenko — CHANGES_REQUESTED** [review]

File: src/test/java/lambda/part1/exercise/Exercise1.java (hunk 1 above)
Comment (elefus) [reply src/test/java/lambda/part1/exercise/Exercise1.java]: Копипаст-копипастом, но править надо)

File: src/test/java/lambda/part1/exercise/Exercise1.java [hunk 2]
```diff
@@ -43,7 +59,18 @@ public void sortPersonsByAgeUsingArraysSortAnonymousComparator() {
     public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator() {
         Person[] persons = getPersons();
 
-        // TODO использовать Arrays.sort
+        Arrays.sort(persons, new Comparator<Person>() {
```
Comment (AleksandrRadchenko) [inline src/test/java/lambda/part1/exercise/Exercise1.java:62]: You can combine comparators and use only one sort here.

**Review by AleksandrRadchenko — CHANGES_REQUESTED** [review]

File: src/test/java/lambda/part1/exercise/Exercise1.java (hunk 2 above)
Comment (turenkoaa) [reply src/test/java/lambda/part1/exercise/Exercise1.java:62]: Thank you, I`ve combined them.

**Review by AleksandrRadchenko — APPROVED** [review]

**Review by ferventdev — APPROVED** [review]

**Review by elefus — APPROVED** [review]

**Pull request merged**